app/views/snacks/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1><%= @snack.name %></h1>
+<p>Snacks:</p>
+<p>Price: <%= number_to_currency(@snack.price) %></p>
+<p>Locations:</p>
+<ul id="snack-locations-<%= @snack.id %>">
+  <% @snack.machines.each do |machine| %>
+    <li>
```

Comment:
Might be good to wrap this block within the each in an id so you can test specific machine info is getting displayed correctly.